Examples/Cocoapods/SpeechToText/Speech/SpeechRecognitionService.swift

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2019, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+import SwiftGRPC
+
+let API_KEY = "YOUR_API_KEY"
```

Comment:
FWIW: you can add a compiler warning (/error) here so that it's easier to find: `#warning("You need to set your API Key")`